### DISCUSSION TITLE
Windows Insider Enrollment and DEF argument

### DISCUSSION THREAD

**Discussion by Arhan2007** [AveYo/MediaCreationTool.bat#49] [Q&A] [answered]
@AveYo 
Hello,
I wanted to know how can I use your Win11 Bypass scripts to enroll an unsupported PC in windows insider dev channel. 

and also how do I add the Def argument to the script

**Comment by AveYo** [comment, marked as answer]
See updated readme with a section about insider builds on unsupported hardware  

As for `def` argument, you have many simple choices! Either:
- rename the script from `MediaCreationTool.bat` to `def MediaCreationTool.bat`    
- call the script with arguments from a powershell / cmd prompt: `MediaCreationTool.bat def`  
- edit the script (line 35) from `rem set /a DEF=1` to `set /a DEF=1`   

Note that you could also just use `MCT Defaults` preset - it will create a vanilla, unmodified media

**Comment by e-t-l** [comment]
@AveYo I have currently installed Win11 using this MCT on an unsupported Win10 PC. When I checked my Windows Insider Settings page I got this error (see screenshot). If I download and run OfflineInsiderEnroll, will that work to change my update channel, or does it have to be run as part of the whole OS installation process? And after running OfflineInsiderEnroll, do you know if I'll still get this Settings error, or if it will reinstate the normal Insider Settings options? Thanks!

![Untitled](https://user-images.githubusercontent.com/40775958/154863114-4759b79b-b665-46b9-bb51-9562ff815dcf.png)

**Reply by AveYo** [reply]
You can run that script and switch the channel freely or stop receiving insider builds at any point.  
Just manage insider settings with the script. 
Once enrolled there are no settings available in the gui but there's also no warning.
